modules/infrastructure-moving-monitoring.adoc

```diff
@@ -7,12 +7,16 @@
 = Moving the monitoring solution
 
 The monitoring stack includes multiple components, including Prometheus, Thanos Querier, and Alertmanager.
-The Cluster Monitoring Operator manages this stack.
-To redeploy the monitoring stack to infrastructure nodes, you can create and apply a custom config map.
+The Cluster Monitoring Operator manages this stack. To redeploy the monitoring stack to infrastructure nodes, you can create and apply a custom config map.
  
 .Procedure
 
-. Save the following `ConfigMap` definition as the `cluster-monitoring-configmap.yaml` file:
+. Edit the `ingresscontroller` resource and change the `nodeSelector` to use the `infra` label.
```

Comment:
`Edit the `ingresscontroller` resource and change the `nodeSelector` to use the `infra` label.`
the change is wrong, change to below is better
`Edit the `cluster-monitoring-config` configMap and change the `nodeSelector` to use the `infra` label`